### ISSUE TITLE
Error on slave

### ISSUE THREAD

**Issue by chrodriguez** [rackspace-cookbooks/mysql-multi#56]
When chef-client is run for the second time on a slave node it fails with:

```
  * mysql_slave_sync[slaves] action create
    
    ================================================================================
    Error executing action `create` on resource 'mysql_slave_sync[slaves]'
    ================================================================================
    
    Mysql2::Error
    -------------
    This operation cannot be performed with a running slave; run STOP SLAVE first
    
    Cookbook Trace:
    ---------------
    /var/chef/cache/cookbooks/mysql-multi/libraries/provider_slave_sync.rb:19:in `query'
    /var/chef/cache/cookbooks/mysql-multi/libraries/provider_slave_sync.rb:19:in `block in <class:MysqlmSlaveSync>'
    
    Resource Declaration:
    ---------------------
    # In /var/chef/cache/cookbooks/mysql-multi/recipes/mysql_slave.rb
    
     48: mysqlm_slave_sync 'slaves' do
     49:   replpasswd node['mysql-multi']['server_repl_password']
     50:   rootpasswd node['mysql-multi']['server_root_password']
     51:   master_ip node['mysql-multi']['master']
     52: end
     53: 

```

**Comment by chrodriguez** [comment]
I don't know if it's correct, but using an only_if block could be used to check if slave is synced?

**Comment by cybermerc** [comment]
Yeah looks like I messed up not putting some type of guard in place here, will try to get this corrected as soon as possible.

**Comment by cybermerc** [comment]
This has been corrected in version 2.1.2 which was published a bit ago.
